UnitTests/TestFixtureLifecycleAttributes/RewriteTestFixtureLifecycleAttributes.cs

```diff
@@ -49,7 +49,7 @@ public void TestSingleAttributeSetUpRewrites (string attribute, string expected)
     [TestCase ("[TestFixtureTearDown ()]", "[OneTimeTearDown ()]")]
     [TestCase ("[TestFixtureTearDown () ]", "[OneTimeTearDown () ]")]
     [TestCase ("[ TestFixtureTearDown () ]", "[ OneTimeTearDown () ]")]

```

Comment:
I actually liked the `[Attribute(...)]` syntax since it showed that `...` was preserved after the transformation, would that make sense here?